src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java

```diff
@@ -75,7 +75,7 @@ public Object invoke(String methodName, Object argument, Type returnType, Map<St
 	 *
 	 * @param methodName the name of the method to invoke
 	 * @param argument the arguments to the method
-	 * @param returnType the return type
```

Comment:
Yeah. I fixed that and also a similar fix on the following method.  (I use IntelliJ to automatically generate the template JavaDoc comment and copy that  order of parameters -- it puts the type parameters last.)